src/main/java/net/glease/ggfab/mte/MTE_AdvAssLine.java

```diff
@@ -440,6 +454,12 @@ protected void drawTexts(DynamicPositionedColumn screenElements, SlotWidget inve
         }));
     }
 
+    @Override
+    public void onPreTick(IGregTechTileEntity aBaseMetaTileEntity, long aTick) {
+        Arrays.fill(mItemCache, NULL);
```

Comment:
I can see this being a problem. Because you would still be calling getStackInSlot every tick? As you always reset the itemCache. You might want this on firstTick.

Reply:
mItemCache's lifespan is exactly one tick.